src/api/entities/confidential/ConfidentialAsset/index.ts

```diff
@@ -81,6 +92,11 @@ export class ConfidentialAsset extends Entity<UniqueIdentifiers, string> {
    */
   public issue: ProcedureMethod<IssueConfidentialAssetParams, ConfidentialAsset>;
 
+  /**
+   * Enable/disable confidential venue filtering for this Confidential Asset and/or set allowed/disallowed Confidential Venues
```

Comment:
```suggestion
   * Enable/disable confidential venue filtering for this Confidential Asset and set the allowed Confidential Venues where it can be exchanged at.
```